cylc/flow/flow_mgr.py

```diff
@@ -30,6 +31,15 @@
 FLOW_NONE = "none"
 
 
+def validate_flow_opt(val):
```

Comment:
Doesn't this semi duplicate the logic in cylc trigger? Surely all use-cases of `--flow` should behave roughly the same, and probably be centralized in `option_parser.py`.